### PR TITLE
styled 404.html

### PR DESCRIPTION
**Pull request by CarolinaGri**
This PR solves issue #124

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -51,17 +62,22 @@
...
 </head>
 
 <body>
   <!-- This file lives in public/404.html -->
   <div class="dialog">
     <div>
-      <h1>The page you were looking for doesn't exist.</h1>
-      <p>You may have mistyped the address or the page may have moved.</p>
+      <h1>Ups, sorry something went wrong... It's an Error 404</h1>
+      <p>The page you were looking for doesn't exist. You may have mistyped the address or the page may have moved. Please try to realod it!</p>
```
Comment (lislis) [inline public/404.html]: realod -> reload

**Comment by CarolinaGri** [comment]
I found another typo "page have--> has moved"- will fix it Monday:)

File: public/404.html [hunk 2]
```diff
@@ -51,17 +82,22 @@
...
 </head>
 
 <body>
   <!-- This file lives in public/404.html -->
   <div class="dialog">
     <div>
-      <h1>The page you were looking for doesn't exist.</h1>
-      <p>You may have mistyped the address or the page may have moved.</p>
+      <h1>Ups, sorry something went wrong... It's an Error 404</h1>
+      <p>The page you were looking for doesn't exist. You may have mistyped the address or the page may have moved. Please try to reload it!</p>
```
Comment (CarolinaGri) [inline public/404.html]: page may have--> has moved

**Comment by lislis** [comment]
glitter

**Pull request merged**